worlds/bomb_rush_cyberfunk/__init__.py

```diff
@@ -0,0 +1,203 @@
+        else:
+            self.item_classification[BRCType.Skateboard] = ItemClassification.progression
+
+        if self.options.starting_movestyle == StartStyle.option_inline_skates:
+            self.item_classification[BRCType.InlineSkates] = ItemClassification.filler
+        else:
+            self.item_classification[BRCType.InlineSkates] = ItemClassification.progression
+        
+        if self.options.starting_movestyle == StartStyle.option_bmx:
+            self.item_classification[BRCType.BMX] = ItemClassification.filler
+        else:
+            self.item_classification[BRCType.BMX] = ItemClassification.progression
+
+
+    def create_items(self):
+        rep_locations: int = 87
+        if self.options.skip_polo_photos:
+            rep_locations -= 18
+
+        self.options.total_rep.round_to_nearest_step()
+        rep_counts = self.options.total_rep.get_rep_item_counts(self.multiworld.random, rep_locations)
+        #print(sum([8*rep_counts[0], 16*rep_counts[1], 24*rep_counts[2], 32*rep_counts[3], 48*rep_counts[4]]), \
+        #    rep_counts)
+
+        pool = []
+
+        for item in item_table:
+            if "REP" in item["name"]:
+                count: int = 0
+
+                if item["name"] == "8 REP":
+                    count = rep_counts[0]
+                elif item["name"] == "16 REP":
+                    count = rep_counts[1]
+                elif item["name"] == "24 REP":
+                    count = rep_counts[2]
+                elif item["name"] == "32 REP":
+                    count = rep_counts[3]
+                elif item["name"] == "48 REP":
+                    count = rep_counts[4]
+
+                if count > 0:
+                    for _ in range(count):
+                        pool.append(self.create_item(item["name"]))
+            else:
+                pool.append(self.create_item(item["name"]))
+
+        self.multiworld.itempool += pool
+
+
+    def get_stage(self, stage: BRCStage, player: int) -> Region:
+        return self.multiworld.get_region(region_names[stage], player)
+
+
+    def create_regions(self):
+        multiworld = self.multiworld
+        player = self.player
+
+        menu = Region("Menu", player, multiworld)
+        multiworld.regions.append(menu)
+
+        for _, n in region_names.items():
+            multiworld.regions += [Region(n, player, multiworld)]
+
+        menu.add_exits({"Hideout": "New Game"})
+
+        for r, e in region_exits.items():
+            exits = {region_names[ex] for ex in region_exits[r]}
+            self.get_stage(r, player).add_exits(exits)
+
+        for index, loc in enumerate(location_table):
+            if self.options.skip_polo_photos and "Polo" in loc["name"]:
+                continue
+            stage: Region = self.get_stage(loc["stage"], player)
+            stage.add_locations({loc["name"]: base_id + index})
+
+        for e in event_table:
+            stage: Region = self.get_stage(loc["stage"], player)
```

Comment:
Wow that's pretty bad lol. Guess I never noticed since they must end up in the "Misc" region and the rules are specific enough for it to work like I wanted anyway. Will fix that though